upgrade/satellite.py

```diff
@@ -120,6 +120,14 @@ def satellite6_upgrade(zstream=False):
     disable_repo_name = ["*"]
     enable_repos_name = ['rhel-{0}-server-rpms'.format(major_ver),
                          'rhel-server-rhscl-{0}-rpms'.format(major_ver)]
+
+    # This statement will execute only until downstream release not become beta.
+    if os.environ.get('NON_BETA_FM_UPGRADE') == 'true':
```

Comment:
```suggestion
    if os.environ.get('DOWNSTREAM_FM_UPGRADE') == 'true':
```

Reply:
It is actually for non-beta version, once beta gets release then the repository name will get reflected in the cdn repo list under beta flag, and at that time there is no need to skip the repository check,  we only use foreman-maintain-beta-flag=1 to use all the downstream repository(if configured) otherwise beta repository.

Reply:
Yeah! it was just a suggestion to rename the env.variable to something better.

Reply:
fixed.